app/gui/src/presenter/searcher.rs

```diff
@@ -221,6 +221,16 @@ impl Model {
         }
     }
 
+    fn set_first_breadcrumb(&self, text: &str) {
+        if let SearcherVariant::ComponentBrowser(browser) = self.view.searcher() {
+            browser.model().list.model().breadcrumbs.set_entry((ImString::new(text).into(), 0));
+        }
+    }
```

Comment:
Instead of "first_breadcrumb" we could refer to it with a meaningful name. For example, "Section name crumb". And make a constant SECTION_NAME_CRUMB_INDEX = 0.